stories/dooboo-ui/Checkbox.stories.tsx

```diff
@@ -27,6 +27,49 @@ const Selected = styled.Text`
   font-weight: bold;
 `;
 
+const defaultCheckboxData: any[] = [
+  {
+    label: 'first',
+    value: 1,
+    selected: true,
+  },
+  {
+    label: 'second',
+    value: 2,
+    selected: false,
+  },
+  {
+    label: 'third',
+    value: 3,
+    selected: false,
+  },
+];
+
+const disabledCheckboxData: any[] = [
```

Comment:
It can be `const defaultData:  CheeckboxItem[]` if you defined `CheckboxItem` interface.